src/test/java/seedu/address/logic/commands/EditCommandTest.java

```diff
@@ -69,15 +69,15 @@ public void execute_someFieldsSpecifiedUnfilteredList_success() {
     }
 
     @Test
-    public void execute_noFieldSpecifiedUnfilteredList_success() {
+    public void execute_noFieldSpecifiedUnfilteredList_failure() {
```

Comment:
May I clarify, is this because of the sorting?

Reply:
The expected behaviour is different from the original AddressBook. The edit Command should throw an error message if the user tries to edit a Person, but the editedPerson and the originalPerson is equal.